src/test/java/org/junitpioneer/jupiter/params/PioneerAnnotationUtilsTests.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.params;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.ResourceLock;
+import org.junitpioneer.jupiter.PioneerException;
+
+@DisplayName("Reflexive copy of PioneerAnnotationUtils")
+@ResourceLock(value = "org.junitpioneer.jupiter.params.PioneerAnnotationUtils")
```

Comment:
I viscerally hate this. I don't even know why. I'm hoping someone smarter comes along and tells me a better solution.

Reply:
Sorry I don't. Afair the `@ResourceLock` is the way how the JUnit Jupiter guys want's you to do.